### DISCUSSION TITLE
Seeed Studio SOM STM32MP157C | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1683] [Q&A]
# Seeed Studio SOM STM32MP157C | Seeed Studio Wiki

Seeed SOM – STM32MP157C

https://wiki.seeedstudio.com/SEEED-SOM-STM32MP157C/

**Comment by dfense** [comment]
Do you provide any examples of a Yocto build with this platform?
